### PR TITLE
Disable secrets protections check in scenario

### PR DESCRIPTION
**Pull request by anuj-apdev**
# Description

Disable secrets protections check as secret protection as already disabled Disable secrets protections check as secret protection as already di…

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Successfully run verify for `rbac-mtls-rhel`

```

PLAY RECAP *********************************************************************
control-center1            : ok=4    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-broker1              : ok=14   changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-broker2              : ok=14   changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-broker3              : ok=14   changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-connect1             : ok=14   changed=4    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-rest1                : ok=4    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
ksql1                      : ok=4    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
schema-registry1           : ok=12   changed=0    unreachable=0    failed=0    skipped=5    rescued=0    ignored=0
zookeeper1                 : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

INFO     Verifier completed successfully.
```

**Test Configuration**:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml [hunk 1]
```diff
@@ -233,8 +233,6 @@ provisioner:
         kafka_connect_replicator_consumer_ssl_cert_path: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
         kafka_connect_replicator_consumer_ssl_key_path: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/{{inventory_hostname}}-key.pem"
         kafka_connect_replicator_consumer_ssl_key_password: keypass
-        kafka_connect_replicator_truststore_storepass: confluenttruststorepass
```
Comment (nsharma-git) [inline molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml:236]: We expect these properties to get overwritten. Why are we removing from molecule files.

Comment (anuj-apdev) [reply molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml:236]: Same property with same name was given twice in the file. That was generating the yamllint warnings.
https://github.com/confluentinc/cp-ansible/blob/353ddc442e4eda5b023a9f257a571a01a0fd76e4/molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml#L217-L232

**Review by utkarsh5474 — APPROVED** [review]
LGTM

File: molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml (hunk 1 above)
Comment (nsharma-git) [reply molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml:236]: Thanks!

**Review by nsharma-git — APPROVED** [review]

**Pull request merged**